src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface MoviesListProps {
+  movies: Movie[];
+}
+
+export const MoviesList: React.FC <MoviesListProps> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => (
+      <MovieCard movie={movie} />
```

Comment:
don't forget to add key by mapping
```suggestion
      <MovieCard key={movie.imdbId} movie={movie} />
```